src/wiki/templates/wiki/base_site.html

```diff
@@ -12,12 +12,6 @@
     <link href="{% static "wiki/bootstrap/css/wiki-bootstrap.min.css" %}" rel="stylesheet">
 
     {% render_block "css" %}
-
-    <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
```

Comment:
The usage of IE6-8 is rather low. https://www.pcworld.com/article/204200/facebook_ends_ie6_support_why_you_should_too.html and that article is 9 years old!

Reply:
Very much agreed!